src/components/projectStatus/ProjectStatusHome.tsx

```diff
@@ -25,21 +26,37 @@ const ProjectStatusHome: React.FC = () => {
       },
     });
 
-  if (projectsLoading) {
+  const [{ loading: tableGroupsLoading, data: tableGroupsData, error: tableGroupsError }] =
+    useAxios({
+      method: "GET",
+      url: apiUrl(Service.EXPO, "/tablegroups"),
+      params: {
+        hexathon: currentHexathon.id,
+      },
+    });
+
+  if (projectsLoading || tableGroupsLoading) {
     return <LoadingDisplay />;
   }
 
-  if (projectsError) {
+  if (projectsError || tableGroupsError) {
     return <ErrorDisplay error={projectsError} />;
   }
 
+  const tableGroupMap = new Map<number, TableGroup>();
+
+  tableGroupsData.forEach((tableGroupItem: TableGroup) => {
+    tableGroupMap.set(tableGroupItem.id, tableGroupItem);
+  });
```

Comment:
can we change this mapping to just id -> name, since all we need is the name

Reply:
With the recent table group changes, this map won't even be used